app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentEvents.kt

```diff
@@ -0,0 +1,23 @@
+package org.simple.clinic.scheduleappointment
+
+import org.simple.clinic.widgets.UiEvent
+import org.threeten.bp.temporal.ChronoUnit
+import java.util.UUID
+
+data class SheetCreated(val initialIndex: Int, val patientUuid: UUID) : UiEvent
+
+data class IncrementAppointmentDate(val currentIndex: Int, val size: Int) : UiEvent {
```

Comment:
The voice of these events is not what we've used so far. The describe "what happened" and not "what to do" :)

`IncrementAppointmentDate` -> `IncrementAppointmentDateClicked`? Or `AppointmentDateIncremented`?

Reply:
Same for the rest of the events